sass/chosen.scss

```diff
@@ -1,6 +1,35 @@
 $chosen-sprite: url('chosen-sprite.png') !default;
 $chosen-sprite-retina: url('chosen-sprite@2x.png') !default;
 
+$chosen-bg-color: #fff;
+$chosen-box-shadow-color: #000;
+$chosen-border-color: #aaa;
+$chosen-group-name-color: #999;
+$chosen-highlighted-color: #3875d7;
+$chosen-highlighted-second-color: #2a62bc;
```

Comment:
I'm not a fan of using two different colors here to represent a gradient. You have gradients in variables below, so why not do the same here. Also, we could clarify that these are background colors. I'd recommend:

```
$chosen-highlighted-bg-color: #3875d7;
$chosen-highlighted-bg-gradient: linear-gradient(#3875d7 20%, #2a62bc 90%);
```